pyproject.toml

```diff
@@ -10,7 +10,7 @@ documentation = "https://sphinx-data-viewer.readthedocs.io/en/latest/"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-sphinx = "^4"
+sphinx = "^5"
```

Comment:
Is there a reason for throwing out Sphinx 4?

I think we should just write `>=4`

Reply:
There's no reason though. I haven't made any changes to the pyproject.toml for the extension